Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -742,13 +744,15 @@
   - BRING ME THE GIRL
   - BRING ME TO LIFE
   - BULLETS
-  - CHILI DOGS
+  - CHILLI DOGS
+  - CHILLY DOGS
   - CORPSES
   - DEODORANT AND A BATH
   - ENOUGH CABBAGES
   - FIVE HUNDRED AND NINETY-NINE US DOLLARS
   - FIVE TEENAGERS WITH ATTITUDE
-  - GODDAMN FUCKING PIECE OF SHIT ASSHOLE BITCH-CHRISTING CUNT-SMUGGLING SWEARING
+  - GODDAMN FUCKING PIECE-OF-SHIT ASSHOLE SWEARING
```

Comment:
"bitch-christing" is a forced, "cunt-smuggling" is not a common insult, and the string is a bit too long to be funny anyway. Also "cunt" is a word that goes from 🤷‍♀️ in Australia to life-ending in the US, so it's probably best to avoid opening that can of worms.